core/src/main/java/com/klaytn/caver/transaction/Transaction.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2021 The caver-java Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.klaytn.caver.transaction;
+
+import com.klaytn.caver.rpc.Klay;
+import com.klaytn.caver.transaction.type_wrapper.*;
+
+/**
+ * Represents a transaction which contains all types of transaction wrapper classes as member variables
+ */
+public class Transaction {
```

Comment:
How about following the same name policy (`~Wrapper` class) ?

Reply:
@jimni1222 
That will be great :)
I'll rename it from `Transaction` to `TransactionWrapper`.
Thanks for your feed back.